test/SimpleClientTests/TransactionSpec.hs

```diff
@@ -20,13 +20,17 @@ exampleAddress1 :: Text
 exampleAddress1 = "2zR4h351M1bqhrL9UywsbHrP3ucA1xY3TBTFRuTsRout8JnLD6"
 
 exampleAccountAddr1 :: Types.AccountAddress
-Right exampleAccountAddr1 = IDTypes.addressFromText exampleAddress1
+exampleAccountAddr1 = case IDTypes.addressFromText exampleAddress1 of
+                        Right addr -> addr
+                        Left err -> error err
 
 exampleAddress2 :: Text
 exampleAddress2 = "4MkK65HrYvMauNTHTuL23wRDKp4VXkCiTpmoWYFtsrZHV3WwSa"
 
 exampleAccountAddr2 :: Types.AccountAddress
-Right exampleAccountAddr2 = IDTypes.addressFromText exampleAddress2
+exampleAccountAddr2 = case IDTypes.addressFromText exampleAddress2 of
+                        Right addr -> addr
+                        Left err -> error err
```

Comment:
Recommend using `fromRight` in all of these address and key parsing.

Reply:
I think that still warrants a comment, and that it may be more natural to err with the message thrown from `addressFromText`?